doc/reference/storage/fcb/fcb.rst

```diff
@@ -0,0 +1,73 @@
+.. _fcb:
```

Comment:
This error is a tricky one. Basically the error message is saying there are two doc labels named “fcb”, so where is the other one? It’s from the doxygen-generated API content. The easiest fix is to change this label to something like “.. _fcb-api:”

Reply:
@dbkinder thanks, fixed  - now works fine